database/index.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/fetcher');
+const Promise = require('bluebird');
+
+var db = mongoose.connection;
+
+let destinationSchema = mongoose.Schema({
+  // TODO: your schema here!
+  id: {
```

Comment:
I think you can update the automatically generated _id property, which will automatically have the required/unique attributes you're looking for - https://mongoosejs.com/docs/guide.html#_id - just a thought, unless you were planning on using the automatically generated _id for something else!